tests/unit/test_progress.py

```diff
@@ -1,14 +1,19 @@
+import logging
 import mock
-from unittest import TestCase
+from dvc.progress import progress, ProgressCallback
 
-from dvc.progress import ProgressCallback
 
+def test_quiet(caplog, capsys):
+    with caplog.at_level(logging.CRITICAL, logger="dvc"):
+        progress._print("something")
+        assert capsys.readouterr().out == ""
 
-class TestProgressCallback(TestCase):
+
+class TestProgressCallback:
     @mock.patch("dvc.progress.progress")
     def test_should_init_reset_progress(self, progress_mock):
```

Comment:
Should we remove the class around it while we are at it?

Reply:
I left it because it is testing the `ProgressCallback` and not `Progress` itself, it brings some clarity, I guess :sweat_smile: